### PR TITLE
Finished the ✨ localStorage ✨ to Bookmark my News

### PR DESCRIPTION
**Pull request by bejanaf**
- added localStorage.js
- added saveToLocal/loadFromLocal to App.js
- refactored News.js to NewsCard.js as component
- Maps added to bookmarks and only the title displayed for testing 
- property handover

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
 import News from './pages/News';
-import Bookmarks from './pages/Bookmarks';
 import Wallet from './pages/Wallet';
-import { useEffect, useState } from 'react';
+import { loadFromLocal, saveToLocal } from './lib/localStorage';
 
 function App() {
   const [articles, setArticles] = useState([]);
+  const [bookmarks, setBookmarks] = useState(loadFromLocal('bookmarks') ?? []);
+  console.log('Ist hier etwas drinnen?', bookmarks);
```
Comment (SabMueller) [inline client/src/App.js]: Please remove this console log :)

File: client/src/components/NewsCard.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    stroke: var(--black);
+    stroke-width: 3px;
+    width: 2.5rem;
+  }
+`;
+
+const FavoriteButton = styled.button`
+  background-color: transparent;
+  border: none;
+  left: 25%;
```
Comment (SabMueller) [inline client/src/components/NewsCard.js]: consider sorting it alphabetically, or put left together with position since they kinda go together :)

**Review by SabMueller — APPROVED** [review]
Looks good to me! :)  ✨ ❤️

File: client/src/components/NewsCard.js (hunk 2 above)
Comment (bejanaf) [reply client/src/components/NewsCard.js]: fixed :)

File: client/src/App.js (hunk 1 above)
Comment (bejanaf) [reply client/src/App.js]: fixed :)

**Comment by bejanaf** [comment]
Thank your very much for your review ✨ :)

**Pull request merged**